lib/utils/string.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns the first string in the data array ending with a null char '\0'
+ * @param {UInt8} data 
+ * @returns the string with the null char
+ */
+var uint8ToCString = function(data) {
+    var index = 0;
+    var curChar = String.fromCharCode(data[index]);
+    var retString = '';
+    while (curChar !== '\0') {
+        retString += curChar;
+        index++;
+        curChar = String.fromCharCode(data[index]);
+    }
+    // Add nullChar
+    retString += curChar;
+    return retString;
+  };
```

Comment:
nit: It looks like this end bracket is not lined up with the function declaration. You also may want to align the module.export statement with the function as well.